signalfx-jvm-agent/README.md

```diff
@@ -0,0 +1,84 @@
+# SignalFx jvm agent
+
+## How to build
+mvn clean install
+
+## How to attach to the java application
+1. build the project
+2. The agent resides in signalfx-jvm-agent.jar in the target directory
+3. When launching the java application include the following parameters:
+java -javaagent:/some/path/signalfx-jvm-agent.jar=path_to_jmx_trans_config \
+-Dcom.signalfx.hostUrl=http://sfproxy-integrations1--aaaa.int.signalfuse.com:28080 \
```

Comment:
I wouldn't put this in the example at all, and instead add it as separate documentation later (like you did)